packages/components/src/components/Dropdown/DropdownMultiInput/DropdownMultiInput.stories.tsx

```diff
@@ -1,18 +1,19 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import { action } from 'storybook/actions';
 
-import DropdownDecorator from '../../../../../../src/storybook/decorators/DropdownDecorator';
+import { DropdownDecorator } from '@ids-sb-decorators/DropdownDecorator';
 import { DropdownMultiInputStateful } from '.';
+import { generateItemsArray } from '@ids-sb-utils/generators';
+
+const DEFAULT_ITEMS_LENGTH = 5;
+const MANY_ITEMS_LENGTH = 50;
+const WRAPPER_HEIGHT_FOR_LONG_LIST = 400;
```

Comment:
These values probably looks familiar because the same are used in single dropdown, but I don't want to share these values between stories.